src/crl.rs

```diff
@@ -19,7 +19,9 @@ use crate::x509::{remember_extension, set_extension_once, DistributionPointName,
 use crate::{Error, SignatureVerificationAlgorithm, Time};
 
 #[cfg(feature = "alloc")]
-use std::collections::HashMap;
+use alloc::collections::BTreeMap as Map;
```

Comment:
As is, there doesn't seem to be much utility in renaming `BTreeMap` to `Map`.

Reply:
I have undone the rename